sqlserver/tests/test_statements.py

```diff
@@ -45,7 +45,7 @@ def dbm_instance(instance_docker):
 def instance_sql_msoledb_dbm(instance_sql_msoledb):
     instance_sql_msoledb['dbm'] = True
     instance_sql_msoledb['min_collection_interval'] = 1
-    instance_sql_msoledb['query_metrics'] = {'enabled': True, 'run_sync': True, 'collection_interval': 0.1}
+    instance_sql_msoledb['query_metrics'] = {'enabled': True, 'run_sync': True, 'collection_interval': 2}
```

Comment:
The `test_statement_metrics_and_plans_windows` test has been super flakey with CI so I tried increasing the collection interval for this, which should  allow us catch metrics in the test